rpc/RPCTransaction.java

```diff
@@ -175,7 +177,6 @@ public void onNext(TransactionProto.Transaction.Res res) {
                 final ResponseCollector collector = collectors.get(requestId);
                 if (collector == null) throw new GraknClientException(UNKNOWN_REQUEST_ID.message(requestId));
                 collector.add(new Response.Ok(res));
-                if (collector.isDone()) collectors.remove(requestId);
```

Comment:
During the flow of streaming calls, it's highly likely that the client will send a `CONTINUE` request that will be picked up by the server after it has already sent a `DONE` message. The server will respond to this `CONTINUE` with a second `DONE`.

When the client received the 1st `DONE` message, it would destroy the response collector. Then, on receiving the 2nd `DONE`, it would throw an exception, saying that the `requestId` is not matched to any response collector.

This change fixes that.

Reply:
Makes sense